README.md

```diff
@@ -45,27 +52,33 @@ Number of samples: 20
 
 | Memory           |   128 |   512 |  1024 |  1769 |
 |------------------|------:|------:|------:|------:|
-| PHP function     | 210ms | 210ms | 210ms | 210ms |
-| HTTP application | 300ms | 300ms | 300ms | 300ms |
+| PHP function     | 408ms | 248ms | 240ms | 228ms |
+| HTTP application | 408ms | 328ms | 320ms | 320ms |
 
 ### Bref 2.x (PHP 8.1)
 
 | Memory           |   128 |   512 |  1024 |  1769 |
 |------------------|------:|------:|------:|------:|
-| PHP function     | 170ms | 170ms | 170ms | 170ms |
-| HTTP application | 250ms | 250ms | 250ms | 250ms |
+| PHP function     | 465ms | 235ms | 210ms | 205ms |
+| HTTP application | 370ms | 280ms | 266ms | 266ms |
+
+### Bref 2.x ARM (PHP 8.0)
```

Comment:
@mnapoli The current benchmarks appear to be comparing the ARM version on PHP 8.0 against PHP 8.1, was this intentional, or just a typo?

Reply:
This is kind of intentional, I'm not sure if that's best tbh. I want to benchmark PHP 8.1 for non-ARM users, because why benchmark an older version when the latest one could be faster.

But I agree this creates a discrepancy.